src/content/extensions/multipleroutes/src/options.js

```diff
@@ -34,7 +34,8 @@ if (!browserSupportsIPHandlingPolicy()) {
 // Saves options.
 function saveOptions() {
   const radios = document.getElementsByName('ip_policy_selection');
-  for (let i = 0; i < radios.length; i++) {
+  let i;
+  for (i = 0; i < radios.length; i++) {
```

Comment:
i was used outside the loop here.

Reply:
i is used below (after breaking out of the loop), line 46

Reply:
yeah. This was ok in the es5 var syntax (which essentially was function scope) but not with the es6 syntax (which does block scope). Now I doubt the extension was ever updated with the code (which tbh doesn't even belong into this repo but... solved the "how can we not have a new repo" process)